Buddies/Activities/View Activity/ViewActivityController.swift

```diff
@@ -180,9 +180,10 @@ class ViewActivityController: UIViewController {
 
     // Gets topics from the root topic store:
     func getTopics(from topicIds: [String]) -> [Topic] {
+        let ids = Array(Set(topicIds))
```

Comment:
Nice!  #13 still will cause problems, but this will hide the problem nicely for now!